apis/python/src/tiledbsoma/_dataframe.py

```diff
@@ -719,6 +720,7 @@ def _build_tiledb_schema(
             dtype=_arrow_types.tiledb_type_from_arrow_type(
                 schema.field(attr_name).type
             ),
+            nullable=metadata.get(bytes(attr_name, "utf-8")) == b"nullable",
```

Comment:
This can also be written as `attr_name.encode("utf-8")`, which (imo) is a little easier to understand